ui/components/MesheryFilterGrid.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import { Grid, Box, Pagination } from "@mui/material";
+import MesheryCard from "./MesheryCard";
+
+const INITIAL_GRID_SIZE = { xl: 4, md: 6, xs: 12 };
+
+function MesheryFilterGridItem({ filter, handleDeploy, handleUnDeploy, handleSubmit, setSelectedFilters }) {
+  const [gridProps, setGridProps] = useState(INITIAL_GRID_SIZE);
+  // const [yaml, setYaml] = useState(filetrs.filter_file);
+
+  return (
+    <Grid item {...gridProps}>
+      <MesheryCard
+        name={filter.name}
+        updated_at={filter.updated_at}
+        created_at={filter.created_at}
+        //  file={filter.filetr_file}
```

Comment:
You don't need to comment the file , handledeploy and handleundeploy @subhradwip01

Reply:
Ya ... actually i was going to uncommet thr file but forgot 😅